hive/src/main/java/com/netease/arctic/hive/CachedHiveClientPool.java

```diff
@@ -73,7 +73,7 @@ private synchronized void init() {
   @Override
   public <R> R run(Action<R, HMSClient, TException> action) throws TException, InterruptedException {
     try {
-      return tableMetaStore.doAs(() -> clientPool().run(action));
+      return tableMetaStore.doAs(() -> clientPool().run(action, true));
```

Comment:
Should we change the `retryByDefault` parameter when constructing `ArcticHiveClientPool`, and keep this `run` method using the default configuration?